### PR TITLE
Rename Plain reporting to Txt.

### LINKED ISSUE #1111: Rename plain report to txt report
Makes more sense to use `txt` so it's symmetric with the file ending. `xml` and `html`.

### PR DESCRIPTION
**Pull request by vanniktech**
Fixes #1111

### CODE REVIEW

**Comment by vanniktech** [comment]
I think this needs one of the many open PRs who fix the master build.

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt [hunk 1]
```diff
@@ -43,10 +43,10 @@ internal class ReportsSpec : Spek({
 			assertThat(xmlReport.path).isEqualTo(Paths.get("/tmp/path1"))
 		}
 
-		it("it should properly parse PLAIN report entry") {
-			val plainReport = reports[1]
-			assertThat(plainReport.kind).isEqualTo(PlainOutputReport::class.java.simpleName)
-			assertThat(plainReport.path).isEqualTo(Paths.get("/tmp/path2"))
+		it("it should properly parse TXT report entry") {
+			val textRepot = reports[1]
```
Comment (pavlospt) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt]: Minor: typo!

File: detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt [hunk 2]
```diff
@@ -25,7 +25,7 @@ internal class ReportsSpec : Spek({
 		val args = arrayOf(
 				"--input", "/tmp/must/be/given",
 				"--report", "xml:/tmp/path1",
-				"--report", "plain:/tmp/path2",
+				"--report", "text:/tmp/path2",
```
Comment (Mauin) [inline detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt]: Should this not be `txt`? I'd hope the test also fails with this current version? (After a rebase, master should be green again).

Comment (vanniktech) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt]: Whops. Yes will fix tomorrow. Had this first but my master was super old and I screwed up when resolving the changes.

Comment (vanniktech) [reply detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/out/ReportsSpec.kt]: Yes. Messed this one up when updating my old stale master branch and resolving the conflicts.

**Comment by arturbosch** [comment]
Looks good! Please add this change to the (not yet created) migrate section of RC10, so I won't forget this.
Also please amend your lat commit message or squash your commits (well I can do it too) to confirm to our contribution guide.

**Comment by vanniktech** [comment]
> Looks good! Please add this change to the (not yet created) migrate section of RC10, so I won't forget this.

Can you elaborate?

> Also please amend your lat commit message or squash your commits (well I can do it too) to confirm to our contribution guide.

You can do that if you select Squash and merge

**Comment by Mauin** [comment]
@vanniktech The migration guide for the next release is currently in the commented out part at the top of: https://raw.githubusercontent.com/arturbosch/detekt/master/docs/pages/changelog.md.

**Comment by vanniktech** [comment]
Should be good to go now. I believe.

**Comment by arturbosch** [comment]
Rebased and merged this manually, thanks!

**Pull request closed without merging**